config/crd/bases/capsule.clastix.io_capsuleconfigurations.yaml

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Why this first line has been removed? Wondering about this because these files are auto-generated.

Reply:
My bad